pages/linux/readelf.md

```diff
@@ -0,0 +1,20 @@
+# readelf
+
+> Displays information about ELF files.
+> More information: <http://man7.org/linux/man-pages/man1/readelf.1.html>.
+
+- Display all information about the ELF file:
+
+`readelf -a {{binary}}`
```

Comment:
Could this be `path/to/binary` please? Same goes for the other examples.

Also, is there a long form of the arguments here (e.g. `-a`, `-e`, etc.) in tldr-pages we prefer to use long options when available to aid clarity.